tools/run-tests.py

```diff
@@ -196,6 +196,15 @@ def get_arguments():
 
 BINARY_CACHE = {}
 
+TERM_EMPTY = '\033[0m'
+TERM_BLUE = '\033[1;34m'
+
+def report_command(cmd, cmd_type):
+    sys.stderr.write('%s%s%s' % (TERM_BLUE, cmd_type, TERM_EMPTY))
```

Comment:
I thought it was intuitive and easy to read this way. I didn't think about the copying though. Maybe the best solution would be to put `\` characters at the end of each line, that would make it copyable. It was intentional to have the first item of the command on the same line, but I think I will break that into a new line as well, that will make copying easier. What do you think about this?

Reply:
I'm not completely convinced about the necessity of the line breaks. But if we are going to add then (for some readability reasons -- but that's somewhat subjective), I'll propose to print it like:
```bash
command \
        --option1 \
        --option2=value2 \
        --option3
```

Note the tabulation of the command line options so that the command is easy to distinguish from its options. Also note the lack of trailing backslash at the end of the last option.

Python can solve all this simply with `sys.stderr.write('%s%s%s\n' % (TERM_BLUE, '\\\n\t'.join(cmd), TERM_EMPTY))`, if I'm not mistaken.